snap/snapcraft.yaml

```diff
@@ -8,7 +8,11 @@ description: |
 
 grade: devel
 confinement: strict
-
+architectures:
+  - build-on: amd64
+  - build-on: arm64
+  - build-on: ppc64el
+  - build-on: s390x
```

Comment:
Hmm, I thought by default all of the supported architectures were built. Has this been a change?

If we do need to be explicit here, then I think the [shorthand form](https://snapcraft.io/docs/architectures#heading--core22) of the architecture arguments may be a bit more clear to the casual observer, e.g.:

```yaml
architectures:
  - amd64
  - arm64
  - ppc64el
  - s390x
```

Reply:
yep, not defining the architectures results in building for 5 architectures (arm64, armhf, amd64, ppc64el, and s390x), this is ok. I'm closing this PR, going with with the implicit behavior is reasonable.